src/components/common/ContentBlock.vue

```diff
@@ -35,7 +35,7 @@
       >
         <b-dropdown-item
           v-for="(item, index) in dropdown.items"
-          :key="item.id"
+          :key="index"
           @click="() => (dropdown.selectedIndex = index)"
```

Comment:
Can we achieve this with just 
```suggestion
          @click="dropdown.selectedIndex = index"
```

Don't usually need to do anonymous functions in Vue as much as React.